lib/path.js

```diff
@@ -270,77 +270,67 @@ const win32 = {
     const code = path.charCodeAt(0);
 
     // Try to match a root
-    if (len > 1) {
-      if (isPathSeparator(code)) {
-        // Possible UNC root
-
-        // If we started with a separator, we know we at least have an absolute
-        // path of some kind (UNC or otherwise)
-        isAbsolute = true;
+    if (len === 1) {
+      // `path` contains just a single char, exit early to avoid
+      // unnecessary work
+      return code === CHAR_FORWARD_SLASH ? '\\' : path;
```

Comment:
shouldn't this use `isPathSeparator(code)` ?

Reply:
It would not change the behavior and just add another check that is not necessary in this case.

If the code is `CHAR_BACKWARD_SLASH` it will still return a backward slash, so we do not have to check that.

Reply:
I could use `isPosixPathSeparator(code)` instead though.

Reply:
I switched to `isPosixPathSeparator`.